casepro/cases/views.py

```diff
@@ -101,10 +101,16 @@ def post(self, request, *args, **kwargs):
                 assignee = request.user.get_partner(self.request.org)
                 user_assignee = request.user
 
```

Comment:
Perhaps just update the comment for this method otherwise it's going to be out of date.